04/style.css

```diff
@@ -0,0 +1,78 @@
+.container {
+  background: #ccc;
+  display: block;
+  width: 30px;
+  height: 100px;
+  margin: 30px auto;
+  margin-top: 10px;
+  padding: 0px;
+}
+
+body {
+  font: 1em arial;
```

Comment:
Don't use complicated font property, separate it to font-family and font size it is a better practice? and never write font family from small letter even if it is still work